### ISSUE TITLE
Unable to open images containers box 

### ISSUE THREAD

**Issue by rkamalakannan** [microsoft/vscode-docker#3165]
Issue Type: <b>Bug</b>

Install extension.
Download an image from the registry (pull)
After image pull, the image is not visible on the image box or container box .

Extension version: 1.16.0
VS Code version: Code 1.59.1 (3866c3553be8b268c8a7f8c0482c0c0177aa8bfa, 2021-08-19T11:56:46.957Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz (8 x 1896)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.88GB (0.45GB free)|
|Process Argv|--crash-reporter-id f9688f08-8474-475c-9d3e-49298f5835e4|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
pythonvspyt639:30300192
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805:30301674
pythonvspyt200:30340761
vscextlangct:30333562
binariesv615:30325510
pythonvssor306:30344512
bridge0708:30335490
vstre464cf:30350173
bridge0723:30353136
vsdyn420:30354109

```

</details>

**Comment by rkamalakannan** [comment]
![image](https://user-images.githubusercontent.com/9931769/130444510-13fe1b69-1a8d-4242-a0dd-873c74493ba8.png)

**Comment by bwateratmsft** [comment]
This looks like it may be a dupe of #3132 which will be fixed in the next release. Do you have Docker installed and running? What happens if you run `docker context ls` in a terminal window?

**Comment by rkamalakannan** [comment]
Yes,  I have the docker installed on my machine. Below is the command output for context ls

docker context ls
NAME                TYPE                DESCRIPTION                               DOCKER ENDPOINT
             KUBERNETES ENDPOINT   ORCHESTRATOR
default *           moby                Current DOCKER_HOST based configuration   npipe:////./pipe/docker_engine 
                                   swarm
desktop-linux       moby                                                          npipe:////./pipe/dockerDesktopLinuxEngine

**Comment by bwateratmsft** [comment]
Does the issue persist if you restart VSCode?

**Comment by rkamalakannan** [comment]
I have restarted VS Code and my machine multiple times. I am unable to run the image which pulled from the registries or I cant see the containers to see files.

**Comment by rkamalakannan** [comment]
I also feel my VS code is slow... How I can increase the speed of loading the extensions at startup of vs code?

**Comment by bwateratmsft** [comment]
We'll need to work out what error is happening. Can you try the latest nightly build? Instructions are available in #1408.

> I also feel my VS code is slow... How I can increase the speed of loading the extensions at startup of vs code?

You can run the command "Developer: Show Running Extensions" to look for what extensions have long activation times.
